### PR TITLE
fix typo

### PR DESCRIPTION
**Pull request by mauilion**
>^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
> Please delete this note before submitting the pull request.
> For 1.12 Features: set Milestone to 1.12 and Base Branch to release-1.12
> Help editing and submitting pull requests:  https://deploy-preview-9510--kubernetes-io-master-staging.netlify.com/docs/contribute/start/#submit-a-pull-request.
> Help choosing which branch to use, see
> https://kubernetes.io/docs/contribute/start#choose-which-git-branch-to-use.
>^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
>

### CODE REVIEW

File: content/en/docs/setup/independent/setup-ha-etcd-with-kubeadm.md [hunk 1]
```diff
@@ -1,7 +1,7 @@
 ---
 reviewers:
 - sig-cluster-lifecycle
-title: Set up a Highly Availabile etcd Cluster With kubeadm
+title: Set up a Highly Available etcd Cluster With kubeadm
```
Comment (neolit123) [inline content/en/docs/setup/independent/setup-ha-etcd-with-kubeadm.md]: thanks for the typo fix. to be fair this should be:
> Set up a High Availability etcd cluster with kubeadm

as per the HA abbreviation.

/assign @Bradamant3

Comment (xiangpengzhao) [reply content/en/docs/setup/independent/setup-ha-etcd-with-kubeadm.md]: +1

@mauilion could you please edit the PR, then LGTM :)

Comment (mauilion) [reply content/en/docs/setup/independent/setup-ha-etcd-with-kubeadm.md]: Corrected. Thanks!

File: content/en/docs/setup/independent/setup-ha-etcd-with-kubeadm.md [hunk 2]
```diff
@@ -1,7 +1,7 @@
 ---
 reviewers:
 - sig-cluster-lifecycle
-title: Set up a Highly Availabile etcd Cluster With kubeadm
+title: Set up a High Availability etcd cluster With kubeadm
```
Comment (neolit123) [inline content/en/docs/setup/independent/setup-ha-etcd-with-kubeadm.md]: sorry, but `With` remained uppercase.
should be lowercase.

**Comment by neolit123** [comment]
/lgtm
thanks

**Comment by mauilion** [comment]
Can someone approve this? @Bradamant3 maybe?

File: content/en/docs/setup/independent/setup-ha-etcd-with-kubeadm.md [hunk 3]
```diff
@@ -1,18 +1,18 @@
 ---
 reviewers:
 - sig-cluster-lifecycle
-title: Set up a Highly Availabile etcd Cluster With kubeadm
+title: Set up a High Availability etcd cluster with kubeadm
```
Comment (zacharysarah) [inline content/en/docs/setup/independent/setup-ha-etcd-with-kubeadm.md]: @mauilion 👋 For future reference: "High Availability" is lower case: _high availability_, _highly available_. See other uses in this article for examples. 

/cc @Bradamant3

**Pull request merged**